### PR TITLE
Ajax Notes and Asset Pipeline assignments

### PR DESCRIPTION
**Pull request by danielhneville**

### CODE REVIEW

File: Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+$("<%= escape_javascript(render @note) %>").appendTo("#notes");
```
Comment (digitalecho) [inline Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb:1]: What does rendering the note itself do? Usually you render view files not objects.

**Review by digitalecho — APPROVED** [review]

File: Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb (hunk 1 above)
Comment (danielhneville) [reply Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb:1]: well, in this case, I think it grabs _note.html.erb, renders that into HTML, then appends that html to the #notes div. That appears to me to be how it's working from what I could glean from documentation and random people on the internet.

Comment (digitalecho) [reply Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb:1]: Did not realize there was a shorthand like that, cool!

**Comment by digitalecho** [comment]
Good Job!

**Pull request merged**